src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts

```diff
@@ -59,62 +59,44 @@ const $vaultAccounts = combine(
   { skipVoid: false },
 );
 
-const $signatoryContacts = combine(
+const $signatories = combine(
```

Comment:
Just form 3 groups of signatories that eliminates any duplication (previously duplicates existed)